include/faker-cxx/word.h

```diff
@@ -23,12 +27,30 @@ namespace faker::word
  * faker::word::sample(5) // "spell"
  * @endcode
  */
-FAKER_CXX_EXPORT std::string_view sample(std::optional<unsigned> length = std::nullopt);
+FAKER_CXX_EXPORT std::string_view sample(std::optional<unsigned> length = std::nullopt,const Locale locale = Locale::en_US);
+
+
+/**
+ * @brief Returns a random .
+ *
+ * @param length The expected length of the .
+ * If 0,it will return a random . 
+ * @param locale The locale. Defaults to `Locale::en_US`. 
+ * @returns Random sample word.
+ *
+ * @code
+ * faker::word::sampleLocale(0) // "protection"
+ * faker::word::sampleLocale(5,Locale::es_AR) // "casas"
+ * @endcode
+ */
+
+FAKER_CXX_EXPORT std::string_view sampleLocale(unsigned length = 0,const Locale locale = Locale::en_US);
```

Comment:
why there are two functions? sample and sampleLocale? it should be only sample, applies to other functions as well

Reply:
solved